src/js/views/patients/worklist/worklist_views.js

```diff
@@ -150,41 +171,81 @@ const ListTitleView = View.extend({
         {{far this}}
       {{/each}}
     </span>
-    {{#if showOwnerDroplist}}
-      <div class="u-text--nowrap">
-        {{formatMessage (intlGet "patients.worklist.worklistViews.listTitleView.listLabels") title=worklistId}}
-      </div>
-      <div data-owner-filter-region></div>
-    {{else}}
-      {{formatMessage (intlGet "patients.worklist.worklistViews.listTitleView.listTitles") title=worklistId team=team owner=owner}}
-    {{/if}}
+    <div data-label-region></div>
+    <div data-owner-filter-region></div>
     <span class="list-page__header-icon js-title-info">{{far "circle-info"}}</span>
-    <div class="u-margin--l-24" data-owner-toggle-region></div>
+    <div data-owner-toggle-region></div>
   `,
   ui: {
     tooltip: '.js-title-info',
   },
   templateContext() {
-    const worklistId = this.getOption('worklistId');
-
     return {
-      team: this.getOption('team').get('name'),
-      owner: this.getOption('owner').get('name'),
-      worklistId: underscored(worklistId),
-      isFlowList: this.getOption('isFlowList'),
-      showOwnerDroplist: this.getOption('showOwnerDroplist'),
-      icons: worklistIcons[worklistId],
+      owner: this.owner.get('name'),
+      worklistId: underscored(this.model.id),
+      icons: worklistIcons[this.model.id],
     };
   },
+  initialize() {
+    const currentClinician = Radio.request('bootstrap', 'currentUser');
+    this.canViewAssignedActions = currentClinician.can('app:worklist:clinician_filter');
```

Comment:
I moved a lot of these `should` checks and the child view rendering from the app to this view and proxy the events back up to the app